letsencrypt/default_server.nginx.jinja

```diff
@@ -0,0 +1,32 @@
+# default servers for nginx
+# this vHosts will catch all request that are not match an other vHosts server_name.
```

Comment:
```suggestion
# To prevent host header injection, this file sets the default_server directive and returns
# HTTP 400 for requests with an unknown hostname in Host header
```